### ISSUE TITLE
Consider implementing live-reload for development

### ISSUE THREAD

**Issue by ioanmo226** [FlowCrypt/flowcrypt-browser#4543]
Currently when I want to see changes (while in development), I have to build project and reload extension frame.
This doesn't seem to be a problem when you want to see changes for so many files.
But it makes me annoying when I want to see few changes (something like changing ui, change font color, margin, etc or just change one line of typescript code)

Here are references.

https://stackoverflow.com/questions/2963260/how-do-i-auto-reload-a-chrome-extension-im-developing
https://github.com/vitejs/vite

Ping: @tomholub

**Comment by tomholub** [comment]
agree. TypeScript generally supports incremental build, so this could be feasible. Have you tried `build-incremental` instead of `build`? It's a bunch faster, and it could be used as the basis for a live-reload mechanism.

**Comment by ioanmo226** [comment]
yeah, i tried. it's faster

**Comment by tomholub** [comment]
You can look into this sometime next month - it would be nice to have & make dev work more pleasant.
